pages/GetAccountLists.generated.ts

```diff
@@ -0,0 +1,45 @@
+import * as Types from '../src/graphql/types.generated';
```

Comment:
Please make all the generated files in a separate commit for the same reasons as above